java/code/src/com/redhat/rhn/common/conf/Config.java

```diff
@@ -318,6 +318,35 @@ public float getFloat(String s, float defaultValue) {
         return val;
     }
 
+    /**
+     * get the config entry for string s
+     *
+     * @param s string to get the value of
+     * @return the value
+     */
+    public Double getDouble(String s) {
+        String val = getString(s);
+        if (val == null) {
+            return null;
+        }
+        return Double.valueOf(val);
+    }
+
+    /**
+     * get the config entry for string s, if no value is found return the defaultValue specified.
+     *
+     * @param s string to get the value of
+     * @param defaultValue Default value if entry is not found.
+     * @return the value
+     */
+    public double getDouble(String s, double defaultValue) {
+        Double val = getDouble(s);
+        if (val == null) {
+            return defaultValue;
+        }
+        return val;
+    }
```

Comment:
Yes. Primitive objects are not nullable and thus I can't return a primitive object above. Since here in all cases I have a primitive object I want to use that. This pattern is also used for `float` in the lines above already.